modules/conductor/src/main/java/org/opencastproject/event/handler/AssetManagerUpdatedEventHandler.java

```diff
@@ -217,13 +217,21 @@ public void handleEvent(final SeriesItem seriesItem) {
             // setting the URI to a new source so the checksum will most like be invalid
             episodeCatalog.setChecksum(null);
           }
+          authorizationService.removeAcl(mp, AclScope.Series);
```

Comment:
Could you maybe move all the code related to the handling with series metadata catalogs up to line 210 so that all that code is at the same place? In particular the handling of the series metadata catalogs should be close together.

Reply:
Ok. My own "todo list" seems to have an other order :)